examples/boundary_conditions.jl

```diff
@@ -0,0 +1,66 @@
+using Test
+using ITensorNumericalAnalysis
+
+using NamedGraphs.NamedGraphGenerators: named_comb_tree
+using ITensors: ITensors, inner
+using ITensorNetworks: ITensorNetwork, ttn, maxlinkdim
+using Random: seed!
+using ITensors: Ops
+
+using UnicodePlots
+
+seed!(1234)
+L = 12
+g = named_comb_tree((2, L ÷ 2))
+lastDigit = 1 - 1 / 2^(L ÷ 2)
+
+s = continuous_siteinds(g; map_dimension=2)
+
+ψ_fxy = cos_itn(s; k=π)
+#ψ_fxy = const_itn(s; c=3, linkdim=3) # note if you use const, need big linkdim
+@show maxlinkdim(ψ_fxy)
+
+maxdim = 20
+cutoff = 0e-16
+@show cutoff
```

Comment:
If the cutoff is zero we can just leave it out of the `map_to_zeros` `kwargs` and not define it

Reply:
Thanks, good catch! That was from the opsum problems, we can switch this to a sensible value now